flink-libraries/flink-table/pom.xml

```diff
@@ -154,6 +154,10 @@ under the License.
 				<artifactId>maven-shade-plugin</artifactId>
 				<executions>
 					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
```

Comment:
If you have tested that a Table API word count can still be executed on the cluster, I'm fine with this change.